site2/docs/adaptors-kafka.md

```diff
@@ -128,7 +128,7 @@ Properties:
 | Config property                         | Supported | Notes                                                                         |
 |:----------------------------------------|:----------|:------------------------------------------------------------------------------|
 | `acks`                                  | Ignored   | Durability and quorum writes are configured at the namespace level            |
-| `auto.offset.reset`                     | Yes       | Will have a default value of `latest` if user does not give specific setting. |
+| `auto.offset.reset`                     | Yes       | Will have a default value of `earliest` if user does not give specific setting. |
```

Comment:
```suggestion
| `auto.offset.reset`                     | Yes       | It uses a default value of `earliest` if you do not give a specific setting. |
```
In technical information, follow these general guidelines for personal pronouns:
• Avoid the first-person pronouns I and we, except in these situations:
° In the question portion of frequently asked questions (FAQs)
° In articles, white papers, or documents that have listed authors and in which the authors describe their own actions or opinions
• Use the second-person pronoun you as much as possible. The subject of an imperative sentence is understood to be you.
• Avoid third-person pronouns that are gender specific.